tools/UpdatePaths/Scripts/66406_turf_updates.txt

```diff
@@ -100,6 +100,7 @@
 /area/command/heads_quarters/hop : /area/station/command/heads_quarters/hop
 /area/command/heads_quarters/hos : /area/station/command/heads_quarters/hos
 /area/command/heads_quarters/rd : /area/station/command/heads_quarters/rd
+/area/command/heads_quarters/qm: /area/station/command/heads_quarters/qm
```

Comment:
Keep in mind that #66406 was merged before we made the QM a Head of Staff, so while it might be fine to re-purpose this UpdatePaths for all mechanical intents, I personally think it would be a lot clearer to create a wholly new file with this change to help.